django/contrib/admin/options.py

```diff
@@ -308,6 +308,21 @@ def get_readonly_fields(self, request, obj=None):
         """
         return self.readonly_fields
 
+    def get_invariable_fields(self, request, obj=None):
```

Comment:
I don't know that "invariable" is the best word -- actually, I didn't know it was a word. I'll try to suggest something if you don't have any alternatives.